### DISCUSSION TITLE
does this support QWERTY 5 row keyboard layout?

### DISCUSSION THREAD

**Discussion by a-gyaanapan** [florisboard/florisboard#1227] [Q&A]
@patrickgold  this is the best keyboard i have ever used. clipboard usage reminded about **native clipboard**.

the only thing am missing or unable to activate is QWERTY 5 row keyboard layout (it saves a lot of time in tricky times, for example pasting). if it is absent, please add it (so i can remove hackers keyboard).

**Comment by patrickgold** [comment]
What exactly do you mean by 5-row QWERTY layout? Is this a layout with a number row on top or does it have special keys?

**Comment by a-gyaanapan** [comment]
@patrickgold https://github.com/klausw/hackerskeyboard provides keyboard similar to what we find on desktop. yes, special keys:  alt, ctrl, del, etc.. peetty much what we find on our US QWERTY layout board.

**Reply by patrickgold** [reply]
Ok FlorisBoard atm does not support a QWERTY keyboard that looks like HackerKeyboard one's, sorry. You can add a number row on top, but arrow keys etc embedded into the character layout will not be supported until I implement layout customization. There are several ways to use arrow keys though, this being in the Smartbar clipboard row and by sliding across the space bar.

**Reply by a-gyaanapan** [reply]
@patrickgold thank you for clarification. i will wait then. once again, thank you.
